services/QuillLMS/spec/controllers/translated_texts_controller_spec.rb

```diff
@@ -34,6 +34,31 @@
       it 'assigns @js_file' do
         expect(assigns(:js_file)).to eq("entrypoints/snippets/translated_text.ts")
       end
+
+      it 'assigns @locales' do
+        create(:translated_text, locale: "es-la")
+        create(:translated_text, locale: "zh-cn")
+        get :index
+        expect(assigns(:locales)).to match_array(["es-la", "zh-cn"])
```

Comment:
Feel free to take or leave this, but I'm always a bit concerned about cases where a string is typed twice.  In this case, it's fine, but I could imagine a typo in a spec update breaking this at some point.  I think I'd prefer something like:
```
it 'assigns @locales' do
  locales = ["es-la", "zh-cn"]
  locales.each { |locale| create(:translated_text, locale:) }
  get :index
  expect(assigns(:locales).to match_array(locales)
end
```
Or some other way of making sure the strings only get typed one time.